pkg/flag/report_flags.go

```diff
@@ -279,7 +280,7 @@ func loadComplianceTypes(compliance string) (spec.ComplianceSpec, error) {
 		return spec.ComplianceSpec{}, xerrors.Errorf("unknown compliance : %v", compliance)
 	}
 
-	cs, err := spec.GetComplianceSpec(compliance)
+	cs, err := spec.GetComplianceSpec(compliance, cache.RealCache{})
```

Comment:
BTW, if the user overrides the cache directory, it will not affect the loading of compliance.

Reply:
I think that's fine because at that point we expect them to manage their own cache and its structure as we cannot make any guarantees on if their cache structure is correct or not.

Reply:
The user only changes the cache directory, but the cache structure remains the same. 

`Global Flags:
      --cache-dir string cache directory`.

Reply:
I see your point - updated https://github.com/aquasecurity/trivy/pull/7179/commits/fe98ea6c5b172af9701979439cbda173680127bb